core/src/rpc.rs

```diff
@@ -382,6 +382,74 @@ impl JsonRpcRequestProcessor {
         Ok(result)
     }
 
+    pub fn get_inflation_reward(
+        &self,
+        pubkey: &Pubkey,
```

Comment:
Oh, we can easily make this `Vec<Pubkey>`.    This'll provide additional utility to the caller, and the work in this method is loading the block from BigTable/rocksdb after which we have all the rewards in hand already

Reply:
Sure, I'll change that.